packages/api/src/brevo-contact/brevo-contact.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from "@nestjs/common";
+
+import { BrevoModule } from "../brevo/brevo.module";
+import { BrevoContactResolver } from "./brevo-contact.resolver";
+import { BrevoContactsService } from "./brevo-contacts.service";
+
+@Module({
+    imports: [BrevoModule],
+    providers: [BrevoContactsService, BrevoContactResolver],
+})
+export class BrevoContactModule {}
```

Comment:
Why is this not part of the brevo module? What are you thoughts behind this decision?

Reply:
Also the brevoModule will contain the brevo api logic for the campaigns & target groups in the future as well.